docs/man/borg-init.1

```diff
@@ -81,6 +76,14 @@ a different keyboard layout.
 You can change your passphrase for existing repos at any time, it won\(aqt affect
 the encryption/decryption key or other secrets.
 .SS Encryption modes
+.sp
+For each repo, the user may select the degree of encryption, and the Hash/MAC 
+algorithm. The table below shows what string to enter, to obtain each combination
+of encryption degree and Hash/MAC algorithm.
+.sp
+Not all modes are backwards compatible with Borg1.0.x. New modes which are not 
+backwards compatible are \fImarked like this\fP 
```

Comment:
why is this better than the old phrasing?

Reply:
When I read "Marked modes are new in Borg 1.1" (with "Marked modes" underlined) I didn't realise at first that the underlining was how they were marked. I assume that whether the text is underlined, bold or italics can depend on the renderer, so you can't say "The underlined modes". Anyway, to me it is clearer to state "marked like this".